flutter_readium/web/_scripts/ReadiumReader.ts

```diff
@@ -91,72 +100,85 @@ class _ReadiumReader {
     }
 
     let preferencesJsonString =
-      !preferencesJson || preferencesJson === 'null' ? '{}' : preferencesJson;
-
-    let preferences = initializePreferencesFromString(preferencesJsonString);
-
-    const configuration: EpubNavigatorConfiguration = {
-      preferences,
-      defaults,
-    };
+      !preferencesJson || preferencesJson === "null" ? "{}" : preferencesJson;
 
     try {
+      // TODO: match native
       this._publication = _ReadiumReader._publications.get(pubId);
       if (!this._publication) {
         const { manifest, fetcher } = await fetchManifest(publicationURL);
         this._publication = new Publication({ manifest, fetcher });
         _ReadiumReader._publications.set(pubId, this._publication);
       }
-
-      if (isAudiobook) {
+      let conformsToArray = this._publication.manifest.metadata.conformsTo;
+
+      if (
+        conformsToArray?.some((profile) => {
+          return profile == Profile.AUDIOBOOK;
+        }) ??
+        false
```

Comment:
I would declare this as a `final isAudiobook = conformsToArray? ...` or better yet, make a helper function on the Publication class, like we do on the Dart and native side.